### PR TITLE
raise invalid input user error for classification task type with continuous label column in feature importances component

### PR DESCRIPTION
**Pull request by imatiach-msft**
raise invalid input user error for classification task type with continuous label column in feature importances component

Copilot summary:

This pull request includes changes to the model monitoring components related to computing feature importance. The most important changes include modifying the import statement in `test_compute_feature_importance.py` to include additional functions and variables for testing, and adding error handling for a specific exception in the `create_lightgbm_model` function in `compute_feature_importance.py`.

Main changes:

* [`assets/model_monitoring/components/tests/unit/test_compute_feature_importance.py`](diffhunk://#diff-609f8646c9f71ec90e0d66e1f2fb943ff2604e663c7273219625750968a96ca6L6-R18): Modified the import statement to include additional functions and variables for testing, and added a new test case to check for correct handling of an `InvalidInputError` exception. [[1]](diffhunk://#diff-609f8646c9f71ec90e0d66e1f2fb943ff2604e663c7273219625750968a96ca6L6-R18) [[2]](diffhunk://#diff-609f8646c9f71ec90e0d66e1f2fb943ff2604e663c7273219625750968a96ca6R171-R185)
* [`assets/model_monitoring/components/src/feature_importance_metrics/compute_feature_importance.py`](diffhunk://#diff-722964c327cbff02d1ff2dfa4cb1875410468efc9643ec8c336aae7abda69415R99-R104): Added error handling for a specific exception in the `create_lightgbm_model` function, imported `InvalidInputError` from `momo_exceptions`, and added a new error message variable. [[1]](diffhunk://#diff-722964c327cbff02d1ff2dfa4cb1875410468efc9643ec8c336aae7abda69415R99-R104) [[2]](diffhunk://#diff-722964c327cbff02d1ff2dfa4cb1875410468efc9643ec8c336aae7abda69415R16) [[3]](diffhunk://#diff-722964c327cbff02d1ff2dfa4cb1875410468efc9643ec8c336aae7abda69415R33-R35)

### CODE REVIEW

File: assets/model_monitoring/components/src/feature_importance_metrics/compute_feature_importance.py [hunk 1]
```diff
@@ -92,7 +97,12 @@ def create_lightgbm_model(X, y, task_type):
         lgbm = LGBMRegressor(boosting_type='gbdt', learning_rate=0.1,
                              max_depth=5, n_estimators=200, n_jobs=1, random_state=777,)
 
-    model = lgbm.fit(X, y)
+    try:
+        model = lgbm.fit(X, y)
+    except ValueError as e:
+        if "Unknown label type: 'continuous'" in str(e):
```
Comment (ycheng35xo) [inline assets/model_monitoring/components/src/feature_importance_metrics/compute_feature_importance.py:103]: Is it also possible to have scenario where target value is numerical but the tasktype is classification? If yes, can we add the validation as well?

Comment (imatiach-msft) [reply assets/model_monitoring/components/src/feature_importance_metrics/compute_feature_importance.py:103]: sorry not sure I understand - this PR is indeed for this bugfix, numerical (continuous) column for classification task type is raising the wrong error, I made it a user error as suggested

**Review by yunjie-hub — APPROVED** [review]

**Review by ycheng35xo — APPROVED** [review]
